diff --git a/node_modules/@expo/cli/build/src/utils/npm.js b/node_modules/@expo/cli/build/src/utils/npm.js
index b861322..289472e 100644
--- a/node_modules/@expo/cli/build/src/utils/npm.js
+++ b/node_modules/@expo/cli/build/src/utils/npm.js
@@ -170,7 +170,7 @@ async function extractNpmTarballAsync(stream, props) {
     transformStream.on("data", (chunk)=>{
         hash.update(chunk);
     });
-    await pipeline(stream, transformStream, _tar().default.extract({
+    await pipeline(stream, transformStream, _tar().extract({
         cwd,
         filter,
         onentry: (0, _createFileTransform.createEntryResolver)(name),
diff --git a/node_modules/@expo/cli/build/src/utils/tar.js b/node_modules/@expo/cli/build/src/utils/tar.js
index 8bf12d8..091c43a 100644
--- a/node_modules/@expo/cli/build/src/utils/tar.js
+++ b/node_modules/@expo/cli/build/src/utils/tar.js
@@ -86,7 +86,7 @@ async function extractAsync(input, output) {
     debug(`Extracting ${input} to ${output} using JS tar module`);
     // tar node module has previously had problems with big files, and seems to
     // be slower, so only use it as a backup.
-    await _tar().default.extract({
+    await _tar().extract({
         file: input,
         cwd: output
     });
